tests/unit/amo/pages/TestUserProfile.js

```diff
@@ -47,7 +47,7 @@ import {
 describe(__filename, () => {
   function defaultUserProps(props = {}) {
     return {
-      display_name: 'Matt MacTofu',
+      name: 'Matt MacTofu',
```

Comment:
These tests are only used for the (read-only) profile page, right?

Reply:
yes

Reply:
This could stay as it was before actually.